src/js/contexts/DataTableColumnContext.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const DataTableColumnContext = React.createContext({});
```

Comment:
```suggestion
export const DataTableColumnsContext = React.createContext({});
```
Can we call it this to align with component name?

Reply:
done